### PR TITLE
Customize footer 

### LINKED ISSUE #6: Customize footer

### PR DESCRIPTION
**Pull request by javierm**
## References

* Closes #6

## Objectives

* Make the footer look like the ones in other similar pages

## Visual Changes

The footer shows contact information and links to transparency and open data

### CODE REVIEW

File: config/locales/custom/es/general.yml [hunk 1]
```diff
@@ -13,3 +13,6 @@ es:
       description: "Todos los datos del Ayuntamiento son tuyos."
     weather:
       title: "El Tiempo"
+  contact:
```
Comment (Senen) [inline config/locales/custom/es/general.yml:16]: I think we are missing the Galician translation for these strings:

```
    phone_1: "%{phone} (Centro de Atención Telefónica)"
    phone_2: "%{phone} (Concello)"
```

File: config/locales/custom/es/general.yml [hunk 2]
```diff
@@ -2,7 +2,9 @@ es:
   sites:
     transparency:
       title: "Transparencia"
+      description: "Obtén cualquier información sobre el Concello de Santiago"
```
Comment (Senen) [inline config/locales/custom/es/general.yml:10]: I think we are missing the Galician translation for this string:
`"Obter información sobre o Concello de Santiago"`

File: config/locales/custom/es/general.yml [hunk 3]
```diff
@@ -2,7 +2,9 @@ es:
...
     open_data:
       title: "Datos abiertos"
+      description: "Todos los datos del Ayuntamiento son tuyos."
```
Comment (Senen) [inline config/locales/custom/es/general.yml:13]: Same here:
`"Todos os datos do Concello son teus."`

File: config/locales/custom/es/general.yml [hunk 4]
```diff
@@ -2,6 +2,8 @@ es:
   layouts:
     header:
       open_gov: Concello de Santiago de Compostela
+    footer:
+      open_source: software libre
```
Comment (Senen) [inline config/locales/custom/es/general.yml:6]: As the Galician translation is the same "software libre" we can avoid to add the translation into the `gl` files.

File: config/locales/es/settings.yml [hunk 5]
```diff
@@ -207,3 +207,13 @@ es:
         max_size_description: "Tamaño máximo permitido al subir un documento (en Megabytes/MB)"
         content_types: "Tipos de documentos permitidos"
         content_types_description: "Selecciona todos los tipos permitidos para los documentos subidos"
+    contact:
```
Comment (Senen) [inline config/locales/es/settings.yml]: I think we should not modify the original Spanish file for settings translations, we could move them to the custom file too `config/locales/custom/es/settings.yml`.

**Review by Senen — CHANGES_REQUESTED** [review]
Hi Javi, I left some comments, all of them regarding translations.

Also noticed that with screen sizes between 640px to 1170px (medium and large Foundation media queries breakpoints) the footer is not rendered properly. 

Here is a patch where I tried to add Foundation column styles to the respective layers so we keep the spacing correct for all screen sizes.

[FooterSpacing.txt](https://github.com/concello-santiago-de-compostela/consul/files/5666754/FooterSpacing.txt)

This another patch is the same but changes the distribution as the following screenshot shows:
[FooterSpacingAndDistribution.txt](https://github.com/concello-santiago-de-compostela/consul/files/5666755/FooterSpacingAndDistribution.txt)

Captura de pantalla 2020-12-09 a las 16 03 49

File: config/locales/custom/es/general.yml (hunk 1 above)
Comment (javierm) [reply config/locales/custom/es/general.yml:16]: Right. We'll ask for translations :+1:.

File: config/locales/es/settings.yml (hunk 5 above)
Comment (javierm) [reply config/locales/es/settings.yml]: You're absolutely right :smile:. Fixed, thanks!

**Review by Senen — APPROVED** [review]

**Review by Senen — APPROVED** [review]

**Pull request merged**